src/configs/base_plugins.c4m

```diff
@@ -231,6 +231,15 @@ Captures metadata from cloud instances.
 """
 }
 
+plugin network {
```

Comment:
ALSO, it JUST occurs to me, that you could already use the proc plugin to get the network routing table, which might help here! That def doesn't require any code changes for this PR, but it might make sense to expose such things as more high-level construct in this plugin over time.

Reply:
in docker container, the proc will have routing table for the docker gateway so prolly something like `172.17.0.1` whereas the thing we actually care about is the next hop over which wont be anywhere in the proc as far as I know

also we already do collect all interfaces from the proc as another key

https://github.com/crashappsec/chalk/blob/697624bac82050ca847c4c779ae36e7ec5dafa6d/src/configs/base_keyspecs.c4m#L4659-L4680